pkgs/development/python-modules/python-sql/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, fetchurl, buildPythonPackage }:
+
+buildPythonPackage rec {
+  name = "python-sql-${version}";
+  version = "0.8";
+  src = fetchurl {
+    url = "mirror://pypi/p/python-sql/${name}.tar.gz";
+    sha256 = "0xik939sxqfqqbpgcsnfjnws692bjip32khgwhq1ycphfy7df3h2";
+  };
+  meta = {
+    homepage = "http://python-sql.tryton.org/";
+    description = "A library to write SQL queries in a pythonic way";
+    maintainers = with lib.maintainers; [ johbo ];
+    license = lib.licenses.bsd3;
+    platform = lib.platforms.all;
```

Comment:
That's correct, meta.platform is inherited from the interpreter. Unless you want to further restrict the platforms there's no need setting it.